### ISSUE TITLE
Can't build on linux

### ISSUE THREAD

**Issue by goujonmael** [haveno-dex/haveno#814]
System : GNU Make 4.4.1
Steps : Default Haveno doc steps
---
```
curl -s "https://get.sdkman.io" | bash
sdk install java 21.0.2.fx-librca
git clone https://github.com/haveno-dex/haveno.git
cd haveno/
git checkout v0.0.18
make #Error 
```
---

Make command give me :
```
make
mkdir -p .localnet
./gradlew build
> Task :cli:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':cli:compileJava'.
> java.lang.NoSuchFieldError: Class com.sun.tools.javac.tree.JCTree$JCImport does not have member field 'com.sun.tools.javac.tree.JCTree qualid'

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.2.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 17s
7 actionable tasks: 7 executed
make: *** [Makefile:15: haveno] Error 1
```

**Comment by woodser** [comment]
Sounds like a JDK or environment problem.

Can you confirm `java --version` shows this:

```
openjdk 21.0.2 2024-01-16 LTS
OpenJDK Runtime Environment (build 21.0.2+14-LTS)
OpenJDK 64-Bit Server VM (build 21.0.2+14-LTS, mixed mode, sharing)
```

**Comment by woodser** [comment]
We only recently updated to JDK 21, so that only applies against the master branch.

For building against v0.0.18, you'd need to use JDK 11.

So this will be fixed on the next release with v0.0.19.
